### PR TITLE
[chore] Don't translate numbers

### PR DESCRIPTION
**Pull request by Frenzie**
Except potentially the likes of 0.7 vs. 0,7 but I'm not convinced that should be done by bothering translators.

Cross-ref to <https://github.com/koreader/koreader/pull/6885>.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6928)

### CODE REVIEW

File: frontend/ui/data/koptoptions.lua [hunk 1]
```diff
@@ -165,7 +165,7 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
                 show_true_value_func = function(str)
                     return string.format("%.1f", str)
                 end,
-                toggle =  {_("0.7"), _("1"), _("1.5"), _("2"), _("3"), _("5"), _("10"), _("20")},
+                toggle =  {_("0.7"), "1", _("1.5"), "2", "3", "5", "10", "20"},
```
Comment (Frenzie) [inline frontend/ui/data/koptoptions.lua:168]: @NiLuJe Thoughts?

**Comment by poire-z** [comment]
> Except potentially the likes of 0.7 vs. 0,7 but I'm not convinced that should be done by bothering translators.

Not convinced either. We have numbers (and formatting with a period) everywhere, so people must be used to see regular numbers, that way.

As for `_("3")`, I would agree to remove them.
Otherwise, we have some work to support [eastern arabic numerals](https://en.wikipedia.org/wiki/Eastern_Arabic_numerals) everywhere :) Pinging @WaseemAlkurdi for opinions: will far far eastern arabs be fine with our shared western arabic numbers ? :)

**Comment by Frenzie** [comment]
> Not convinced either. 

Thanks, I'll go ahead and merge this quickly then so that the rest of the translators aren't bothered by this like the first half dozen.

**Comment by NiLuJe** [comment]
I tend to vastly prefer the `.` as a decimal separator, even though it's supposed to be `,` over here, so I'm admittedly a bit of an odd duck on that front ;).

TL;DR: Yes ^^.

**Comment by WaseemAlkurdi** [comment]
@poire-z Apologies, I'm too late ... 
Yes, if it takes too much effort to implement eastern Arabic numerals, then western Arabic (i.e. Arabic) numerals would be just fine, both systems are used around here.
I'd be curious to hear what speakers of Farsi (Persian) think about this, since they use the eastern numerals much more than speakers of Arabic or Kurdish.
